src/main/java/com/google/cloud/tools/action/app/GenConfigAction.java

```diff
@@ -29,18 +29,21 @@
 /**
  * Generates missing configuration files.
  */
-public class GenConfigAction extends AppAction {
+public class GenConfigAction {
 
   private static Logger logger = Logger.getLogger(GenConfigAction.class.getName());
   private GenConfigConfiguration configuration;
+  private CloudSdkExecutor sdkExec;
 
-  public GenConfigAction(GenConfigConfiguration configuration) {
+  public GenConfigAction(GenConfigConfiguration configuration, CloudSdkExecutor sdkExec) {
```

Comment:
I tried very hard to avoid this. :) I thought the aim of this work was to offer a gcloud-agnostic interface, which we're not doing here.